vonk/configuration/appsettings.rst

```diff
@@ -354,6 +366,8 @@ Subscriptions
 
 See :ref:`feature_subscription`.
 
+.. _fhir_capabilities:
+
```

Comment:
further down in this file at PipeLineOptions: not a must, but I think it would push people in the right direction if we included 'Vonk.Fhir.R4" in the unfinished lists (before //etc.)

Reply:
Added